### PR TITLE
Introduce BoundPeerExtensions and .QueryAppProtocolVersion()

### PR DESCRIPTION
**Pull request by longfin**
~This PR introduces `NetMQTransport.QueryAppProtocolVersion()` to get current APV of specific node via `Ping` message.~

This PR contains two changes

1. it moves `ITransport` and `NetMQTransport` to a new `Libplanet.Net.Transports` namespace.
2. it introduces `Libplanet.Net.Transports.BoundPeerExtensions` and `.QueryAppProtocolVersion()`  to get current APV of specific node via `Ping` message.

### CODE REVIEW

**Comment by limebell** [comment]
How about introduce new method to `ITransport`?

**Comment by longfin** [comment]
> How about introduce new method to `ITransport`?

It's a good idea, but `.QueryAppProtocolVersion()` is static, and interfaces can't have static methods because they're contracts for instances, not types. it can be some kind of code smell or evidence of the wrong abstraction but I don't have any idea for it at this time 🤔

**Review by limebell — APPROVED** [review]

**Review by dahlia — DISMISSED** [review]

**Review by riemannulus — APPROVED** [review]

**Comment by longfin** [comment]
Changed the approach. PTAL @libplanet  🙏

File: CHANGES.md [hunk 1]
```diff
@@ -35,7 +38,11 @@ To be released.
 
 [#1219]: https://github.com/planetarium/libplanet/pull/1219
 [#1228]: https://github.com/planetarium/libplanet/pull/1218
+<<<<<<< HEAD
 [#1234]: https://github.com/planetarium/libplanet/pull/1234
+=======
+[#1235]: https://github.com/planetarium/libplanet/pull/1235
+>>>>>>> 5efa0c75... Introduce BoundPeerExtensions
```
Comment (limebell) [inline CHANGES.md:45]: Conflict signature included, please remove this ;)

**Review by limebell — CHANGES_REQUESTED** [review]

**Review by limebell — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Pull request merged**